code-challenges.test.js

```diff
@@ -15,42 +17,147 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
+// describe("arrayFibz", () => {
+//     it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence", () => {
+//         const fibLength1 = 6
+//         // // Expected output: [1, 1, 2, 3, 5, 8]
+//         const fibLength2 = 10
+//         // // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+//         expect(arrayFibz(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+//         expect(arrayFibz(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+
+//     })
+// })
```

Comment:
Great job with the test!  I saw your error over the weekend and realize I may have confused you with the quotes comment.  Glad Corey helped you out and you were able to make progress!  Much improved!